pandas/tests/frame/test_to_csv.py

```diff
@@ -874,6 +874,13 @@ def test_to_csv_index_no_leading_comma(self):
         expected = tm.convert_rows_list_to_csv_str(expected_rows)
         assert buf.getvalue() == expected
 
+    def test_to_csv_multi_index(self):
+        # see gh-26303
```

Comment:
don’t actually write a file as that leaves an artifact
either use a context manager (see other routines) or write to StringIO

in either case compare the result

Reply:
Thanks for this comment @jreback, resolved in c5d36b9d8b1a7935448a0581f6fd9105c4a3ad57